<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotateScenarioDialog</class>
 <widget class="QDialog" name="RotateScenarioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>159</height>
   </rect>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mAngleGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="mAngleLineEditStatusLabel">
        <property name="toolTip">
         <string>This allows the user to see the reason for a color change of the line edit text.</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="UtQtAngleValueEdit" name="mAngleLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>This allows the user to type in the angle (value and unit) by which to rotate.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="mAngleLabel">
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="4">
       <widget class="QDial" name="mAngleDial">
        <property name="toolTip">
         <string>This allows the user to input an integral angle by which to rotate.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MapUtils::ScenarioTransformation::DialogMenuAndButtonsWidget" name="mDialogMenuAndButtons" native="true"/>
   </item>
   <item>
    <spacer name="mSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapUtils::ScenarioTransformation::DialogMenuAndButtonsWidget</class>
   <extends>QWidget</extends>
   <header>DialogMenuAndButtonsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UtQtAngleValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
